float64[7] tau_d
float64[7] tau_d_fli
float64[7] tau_J
#
float64[7] q
float64[7] dq
float64[7] q_d
float64[7] dq_d
float64[7] qError
float64[7] dqError

float64[6] X
float64[6] dX
float64[6] X_d
float64[6] dX_d
float64[6] Xerror
float64[6] dXerror

float64[3] pos
float64[3] pos_d
float64[3] ori
float64[3] ori_d
float64[3] pos_error
float64[3] ori_error

float64[6] F_ext0
float64[6] F_extK
float64[6] F_c
float64 depthDistance
float64 VirtualForce

float64[7] D
# float64[3] K
